### ISSUE TITLE
Selecting NVPTX arch does not work

### ISSUE THREAD

**Issue by gjvdbraak** [clang-omp/libomptarget#4]
Selecting an NVPTX architecture (e.g. sm_30) doesn't work with cmake, the default is always used. To fix this, change line 36 in `libomptarget/DevRTLs/nvptx/CMakeLists.txt` from

``` cmake
if(DEFINED ${OMPTARGET_NVPTX_SM})
```

to: (remove curly brackets and $ around OMPTARGET_NVPTX_SM)

``` cmake
if(DEFINED OMPTARGET_NVPTX_SM)
```

PS. maybe add documentation about this feature to a README.md file, something like:

```
The CUDA architecture for the cuda-RTL can be set using:
    cmake -DOMPTARGET_NVPTX_SM=30 ../
```

**Comment by gjvdbraak** [comment]
Will also be solved by merge request #5.

**Comment by gjvdbraak** [comment]
Will also be solved by merge request #5.

**Comment by sfantao** [comment]
Hi gjvdbraak,

Please verify the issue.

Thanks!
Samuel

**Comment by gjvdbraak** [comment]
Issue can be closed, it has been resolved after merging with #5.
